latex/Research_Strategy.tex

```diff
@@ -213,8 +219,8 @@
 %b.1 Figure showing progression in hydration free energy accuracy across a couple of SAMPLs or comparing two SAMPLs?
 %
  %Fennell SEA, etc.
-Shifts in protonation state and tautomer proved particularly important in the recent SAMPL5 $\log D$ challenge[refs], as they presumably will be for protein-ligand binding as well.
-Host-guest binding studies have also been particularly important[ref benchmark sets],
+Shifts in protonation state and tautomer proved particularly important in the recent SAMPL5 $\log D$ challenge [refs], which functions as a much more tractable venue for exploring the role these effects play in more complex protein-ligand binding.
```

Comment:
I'll have to hit this sentence up in revisions -- I don't like the "much more tractable venue" phrasing at all but I get what you're trying to do, so I need to rephrase. (It's too late in the day for me to do a good job coming up with new phrasings at this point, sadly.)